README.md

```diff
@@ -74,7 +74,7 @@ Make a note of the `token` value in the response, as you will need it later on w
 
   ### Cross device URL ###
 
-  This is an enterprise feature that must be enabled for your account before it can be used. Once enabled you will be able to specify your own custom url that the cross-device flow will redirect to instead of the Onfido default. To use this feature generate a SDK token as shown below and use it to start the SDK.
+  This is a premium enterprise feature that must be enabled for your account before it can be used. Once enabled you will be able to specify your own custom url that the cross-device flow will redirect to instead of the Onfido default. To use this feature generate a SDK token as shown below and use it to start the SDK.
```

Comment:
I think that just leaving this here might still generate some confusion. We should either remove it or add info about how to enable the feature. Maybe adding something like this would help? `For more information, please contact your account manager`

Reply:
Talked to Steve and updated this. Thanks